### PR TITLE
User deletion/deactivation

### PR DESCRIPTION
**Pull request by franmux01**

### CODE REVIEW

**Comment by franmux01** [comment]
Activate User

File: Services/UserService.cs [hunk 1]
```diff
@@ -299,6 +306,124 @@ public async Task<ModifyUserResponse> ModifyUser(ModifyUserRequest rq, ClaimsIde
...
+            {
+                return new DeleteUserResponse
+                {
+                    Success = false,
+                    Message = ex.Message,
+                };
+            }
+        }
+
+        public async Task<ActivateUserResponse>ActivateUser(ActivateUserRequest rq, ClaimsIdentity ? identity)
```
Comment (MaxiPepa) [inline Services/UserService.cs]: Add space before and after Activate user, remove space in the ? on ClaimsIdentity

File: Services/UserService.cs [hunk 2]
```diff
@@ -299,6 +306,124 @@ public async Task<ModifyUserResponse> ModifyUser(ModifyUserRequest rq, ClaimsIde
...
+            if (user == null)
+            {
+                return new ActivateUserResponse
+                {
+                    Success = false,
+                    Message = "User not found",
+                };
+            }
+
+            if (user.Inactive == false)
```
Comment (MaxiPepa) [inline Services/UserService.cs]: Change for if (!user.Inactive)

File: Controllers/UserController.cs [hunk 3]
```diff
@@ -81,6 +81,26 @@ public async Task<IActionResult> ModifyUser([FromBody] ModifyUserRequest rq)
             else
                 return BadRequest(rsp);
         }
+
+        [HttpPost]
+        public async Task<IActionResult> DeleteUser([FromBody] DeleteUserRequest rq)
+        {
+            var rsp = await _userService.DeleteUser(rq, HttpContext.User.Identity as ClaimsIdentity);
```
Comment (MaxiPepa) [inline Controllers/UserController.cs:88]: Add space after rsp

File: Controllers/UserController.cs [hunk 4]
```diff
@@ -81,6 +81,26 @@ public async Task<IActionResult> ModifyUser([FromBody] ModifyUserRequest rq)
...
+            if(rsp.Success)
+                return Ok(rsp);
+            else
+                return BadRequest(rsp);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> ActivateUser([FromBody] ActivateUserRequest rq)
+        {
+            var rsp = await _userService.ActivateUser(rq, HttpContext.User.Identity as ClaimsIdentity);
```
Comment (MaxiPepa) [inline Controllers/UserController.cs:99]: Add space after rsp

**Review by MaxiPepa — CHANGES_REQUESTED** [review]

**Review by MaxiPepa — APPROVED** [review]

**Pull request merged**